src/algebraic_geometry/presheafed_space.lean

```diff
@@ -155,6 +157,77 @@ def forget : PresheafedSpace C ⥤ Top :=
 
 end
 
+section iso
+
+variables {X Y : PresheafedSpace C}
+
+/--
+An isomorphism of PresheafedSpaces is a homeomorphism of the underlying space, and a
+natural transformation between the sheaves.
+-/
+@[simps hom inv]
+def iso_of_components (H : X.1 ≅ Y.1) (α : H.hom _* X.2 ≅ Y.2) : X ≅ Y :=
+{ hom := { base := H.hom, c := α.inv },
+  inv := { base := H.inv,
+    c := to_pushforward_of_iso H α.hom },
+  hom_inv_id' := by { ext, { simp, erw category.id_comp, simpa }, simp },
+  inv_hom_id' :=
+  begin
+    ext x,
+    induction x using opposite.rec,
+    simp only [comp_c_app, whisker_right_app, to_pushforward_of_iso_app,
+      nat_trans.comp_app, eq_to_hom_app, id_c_app, category.assoc],
+    erw [←α.hom.naturality],
+    have := nat_trans.congr_app (α.inv_hom_id) (op x),
+    cases x,
+    rw nat_trans.comp_app at this,
+    convert this,
+    { dsimp, simp },
+    { simp },
+    { simp }
+  end }
+
+/-- Isomorphic PresheafedSpaces have homeomorphic topological spaces. -/
+def base_iso_of_iso (H : X ≅ Y) : X.1 ≅ Y.1 :=
```

Comment:
Note that this is just `(forget C).map_iso`

Reply:
Oops

Reply:
I'm not sure, it depends on whether you use `base_iso_of_iso` a lot and whether you think having this extra definition makes it more readable than just using `(forget C).map_iso` directly.